conf/documentation.conf

```diff
@@ -1753,6 +1753,12 @@ description=<<EOT
 Interval at which PacketFence checks the health state of the cluster and attempts to resolve any detected conflicts
 EOT
 
+[maintenance.option_82_task_interval]
```

Comment:
s/option_82_task_interval/option82_query_interval ?

IMO, "task" is a bit ... generic...